ballista/CHANGELOG.md

```diff
@@ -19,6 +19,289 @@
 
 # Changelog
 
+## [ballista-0.7.0](https://github.com/apache/arrow-datafusion/tree/ballista-0.7.0) (2022-05-12)
+
+[Full Changelog](https://github.com/apache/arrow-datafusion/compare/7.1.0-rc1...ballista-0.7.0)
```

Comment:
@alamb This looks better now although still slightly confusing because there is also a reference to `7.1.0-rc1` but I think we can live with that.